package.json

```diff
@@ -18,7 +18,8 @@
     "lint": "eslint ./src",
     "prepublishOnly": "npm run prettier:fix && npm run lint && npm run test",
     "prettier:ci": "prettier --list-different \"src/**/*.[jt]s\" --parser typescript",
-    "prettier:fix": "prettier --write \"./src/**/*.[jt]s\" --parser typescript"
+    "prettier:fix": "prettier --write \"./src/**/*.[jt]s\" --parser typescript",
```

Comment:
Can we add this to our `checks.sh` script? then it will run via our working circle-ci

Reply:
Sure!
Will remove the workflow file then